spring-cloud-skipper-client/src/main/java/org/springframework/cloud/skipper/client/SkipperClient.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.skipper.client;
+
+import org.springframework.cloud.skipper.domain.AboutInfo;
+
+/**
+ * The main client side interface to communicate with the Skipper Server.
+ *
+ * @author Mark Pollack
+ */
+public interface SkipperClient {
+
+	static SkipperClient create(String baseUrl) {
```

Comment:
I kinda like this pattern, how about we just add another create method and then change the spots in the code that call 'new DefaultSkipperClient' ?